build.sbt

```diff
@@ -1078,8 +1078,8 @@ lazy val `aws-sandbox` = projectMatrix
       "-Wconf:cat=deprecation:silent"
     ),
     smithy4sDependencies ++= Seq(
-      "com.disneystreaming.smithy" % "aws-cloudwatch-spec" % "2023.02.10",
-      "com.disneystreaming.smithy" % "aws-ec2-spec" % "2023.02.10"
+      "com.disneystreaming.smithy" % "aws-cloudwatch-spec" % "2023.09.22",
```

Comment:
Just brought these up to match Dependencies.scala's `AwsSpecSummary`, but this is not the latest version. The latest is a bit wonky, we should deal with that separately.